mmtk/src/vm_metadata/constants.rs

```diff
@@ -35,19 +31,18 @@ pub(crate) const FORWARDING_BITS_METADATA_SPEC: VMLocalForwardingBitsSpec =
 /// 1 bit per object
 #[cfg(feature = "mark_bit_in_header")]
 pub(crate) const MARKING_METADATA_SPEC: VMLocalMarkBitSpec =
-    VMLocalMarkBitSpec::in_header(FORWARDING_BITS_OFFSET);
+    VMLocalMarkBitSpec::side(match LOS_METADATA_SPEC.get_spec() {
```

Comment:
This needs to be updated to the new API.

Reply:
Fixed